Assignment2/aelxfun200-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,38 @@
+{
+	"@context": {
+			"base": "http://www.oeg-upm.net/ontologies/class/",
+			"vocab_1": "http://www.oeg-upm.net/ontologies/class/computer#",
+			"vocab_2": "http://www.oeg-upm.net/ontologies/class/sensor#",
+			"vocab_3": "http://www.oeg-upm.net/ontologies/class/measurement#",
+			"vocab_4": "http://www.oeg-upm.net/ontologies/class/user#",
+			"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+			"name": "rdfs:label",
+			"base_case": "base:base_case", 
+			"comp": "vocab_1:vocab_1_case",
+			"owner": "vocab_4:vocab_4_case",
+			"sensor": "vocab_2:vocab_2_case",
+			"temperatura": "sensor:Temp",
+			"fech_hora": "sensor:AtTime"
+	},
```

Comment:
The attributes "includes", "hasMeasurement" are missing. With includes you have to specify the relationship between Class01 and Sensor029 and Computer101. 
Please, check your data definitions.